### ISSUE TITLE
'aria-expanded' / 'aria-labelledby' attributes not well supported on elements with no 'role' attribute

### ISSUE THREAD

**Issue by MattSidor** [AlaskaAirlines/auro-dropdown#133]
## Describe the bug

The `<auro-dropdown>` element's [`aria-expanded`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) attribute and the `<div class="triggerContent">` element's [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) attribute are being flagged by axe-core 4.4.2 (Chrome extension) since the same elements where these attributes are placed have no [`role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) attribute applied.

According to the tool, these attributes may be ignored by screen readers if there is no `role` attribute.

## To Reproduce

1. Go to https://auro.alaskaair.com/components/auro/dropdown
1. Run [Axe DevTools extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
1. Note the two violations of rule ID [`aria-allowed-attr`](https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr) within each `<auro-dropdown>` element.

## Expected behavior

No Axe rule violations.

## Desktop (please complete the following information):

 - OS: macOS
 - Browser: Chrome 
 - Version: 103

## Additional context

This issue was identified while investigating accessibility test failures in [auro-select](https://github.com/AlaskaAirlines/auro-select). An issue was opened there (https://github.com/AlaskaAirlines/auro-select/issues/77), but making the fixes in auro-dropdown instead should solve the same issues in auro-select and [auro-combobox](https://github.com/AlaskaAirlines/auro-combobox), which both use auro-dropdown.
